### DISCUSSION TITLE
Student benefits

### DISCUSSION THREAD

**Discussion by Fredricka23** [community/community#29436] [GitHub Education]
Please help. What type of proof should I upload for GitHub to approve my request for student benefits (packages) ? I have been rejected 3 times

**Comment by delosemre** [comment]
**To qualify for student benefits, you must:**

- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution.
- Have a verifiable school-issued email address or upload documents that prove your current student status.
- Have a GitHub user account.
- Be at least 13 years old.

In short, you must be studying at a university and have a university e-mail address.

**Comment by manasesjesus** [comment]
You need to activate location services on your browser and go to this webpage: https://education.github.com/discount_requests/student_application

Providing your current location helps GitHub to verify your affiliation with your chosen school/university. You are far less likely to be verified if you choose not to share your location. 

Provide your school/university email. Personal or work emails aren't allowed. It must be from an educational institution.

Provide the other requested data, and that's it. You'll get the GitHub student benefits for two years. After that, if you're still a student, you can renew it.

**Comment by softwaredevelop** [comment]
> Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.

https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack#requirements
